### ISSUE TITLE
Handle BNode object type

### ISSUE THREAD

**Issue by samuell** [rdfio/RDFIO#35]
This happens when trying to import certain RDF/XML constructs:

![selection_195](https://cloud.githubusercontent.com/assets/125003/25334414/d65f6424-28ee-11e7-9245-55b33914b567.png)

([Relevant piece of code](https://github.com/rdfio/RDFIO/blob/91d4a1/classes/parsers/RDFIO_ARC2ToWikiConverter.php#L74-L87))

Some links with relevant info:

- [JSON triple sets (port of ARC2 triple set structure)](https://www.w3.org/wiki/JSON_Triple_Sets)

**Comment by samuell** [comment]
After adding a switch case for bnode (adfe9de43eca605a994daa8e7553616742444307), one can now see how badly RDF/XML import works.

Trying to import this:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ns0="http://www.recshop.fake/cd#"
         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">

  <rdf:Description rdf:about="http://www.recshop.fake/cd/Empire Burlesque">
    <ns0:artist>Bob Dylan</ns0:artist>
    <ns0:country>
      <rdf:Description rdf:about="http://www.countries.org/onto/USA">
        <rdfs:label>USA</rdfs:label>
      </rdf:Description>
    </ns0:country>

    <ns0:company>Columbia</ns0:company>
    <ns0:price>10.90</ns0:price>
    <ns0:year>1985</ns0:year>
    <rdf:type>Album</rdf:type>
  </rdf:Description>

  <rdf:Description rdf:about="http://www.recshop.fake/cd/Hide your heart">
    <ns0:artist>Bonnie Tyler</ns0:artist>
    <ns0:country>UK</ns0:country>
    <ns0:company>CBS Records</ns0:company>
    <ns0:price>9.90</ns0:price>
    <ns0:year>1988</ns0:year>
    <rdf:type>Album</rdf:type>
  </rdf:Description>

  <rdf:Description rdf:about="http://www.countries.org/onto/Albums">
    <rdfs:subClassOf rdf:resource="http://www.countries.org/onto/MediaCollections"/>
  </rdf:Description>

</rdf:RDF>
```

... results in this:

![selection_196](https://cloud.githubusercontent.com/assets/125003/25334878/e4abd736-28f0-11e7-9226-a3b7e58050a8.png)

**Comment by samuell** [comment]
@zidama It seems this was easier to fix than I though! Need to test a bit more though. Will get back with more info as soon as I have tested a bit more.

**Comment by samuell** [comment]
This is now fixed in [v2.3.0](https://github.com/rdfio/RDFIO/releases/tag/v2.3.0). (Ping @zidama)
